### ISSUE TITLE
Please change the name to a more professional & international one

### ISSUE THREAD

**Issue by andrew712-1** [SchildiChat/schildichat-graphics#3]
Please change the name of the app, it's too long, and hard to pronounce especially for russians. We need a shorter and easier to pronounce (internationally) name.

I love your app, but I can't recommend it to anyone because of this  issue.

A few ideas I have:

Turchat (Turtle + chat)
Chatur (Chat + turtle)
T-chat (Turtle chat)

Check this other issue regarding app logo https://github.com/SchildiChat/schildichat-graphics/issues/2

**Comment by SpiritCroc** [comment]
Good idea to split up the issues, since these are two different things.
My point I wrote in the other issue still stands:

> A name change to a more international one would need to be somehow related to the current name, I don't think a full re-brand is worth the effort. If you have a really good idea, I might consider it, but probably won't happen anytime soon.
> 
> I see that the name might look hard to pronounce, but I don't think it actually is hard to pronounce if you know how to. For the same pronunciation in English, you might write something like "Shilldy-Chat", I guess.
> 
> "Too long" - well, it's three syllables, so same as "Element", but way easier to search for in the Internet. I don't see the need to make it any shorter, while risking being less searchable.

Btw, feel free to call it "Schildi" if "SchildiChat" is too long for you.

My opinion about a possible re-branding is actually stronger than about changing the icon. I don't think that changing the name a good idea or necessary right now. You're still allowed to write proposals, and if there's one that I consider brilliant / possibly worth the effort of changing the name, I'll let you know - but in general, I'm not really interested in discussing other names.

**Comment by notramo** [comment]
I'm sure I won't recommend it to friends while the current name is used. It's hard to pronounce in languages other than German, and hard to find. This name (and logo) ruins what the app gains in terms of UX by changing login screen, chat list and other features.

What about **TurtleChat**?
It's easy to understand, spell, and search, and carries some of the ideas of the original name.

**Comment by Duvio** [comment]
I fully agree. I thought about introducing Matrix in our organization, but taking into account that we are closely tied to the government it would look incredibly goofy to have all our official internal communication run through an app that has a turtle as icon and seems like it is marketed towards children. Sadly, all the other Android clients are either not mature enough or extremely buggy.

**Comment by SKlein-1428** [comment]
Throwing in my tuppence, I think if this is a big enough deal for you specifically wrt professionalism, your priorities are fundamentally flawed. I'm tired of things being sterilized to appeal to asinine corporate stylings.

I can't comment on the ease of pronunciation or spelling for international accessibility - as an English speaker, Germanic languages are all relatively similar to my own - but "we'd all look silly if we used a turtle chat so we're not going to use the superior technology due to optics" is such backwards logic I can't actually grasp it.
